### ISSUE TITLE
Drone build failed

### ISSUE THREAD

**Issue by xoxys** [drone/drone-plugin-index#196]
... maybe because cached hugo image in drone pipeline is a bit outdated

**Comment by tboerger** [comment]
I have merged your PR, let's see if that fixes the build.

**Comment by xoxys** [comment]
It does not....

**Comment by xoxys** [comment]
Ok I found the reason and a solution. 

Reason:
`content/banzaicloud/drone-kaniko/index.md` use hugo's built-in `rel` shortcode to dynamically link to `drone-plugins/drone-docker`. The drone pipeline runs a script to generate copies of all topics for missing languages.

The filenames contain `*.gen.*` (maybe to determine if a language file was manually created or auto-generated) and looks like  `content/Comcast/drone-cloudfoundry/index.gen.es.md`. This dot-notation generates a strange subfolder in the urls `http://plugins.drone.io/fr/robertstettner/drone-cloudformation/index.gen/` and `drone-plugins/drone-docker` is no longer a valid reference for other languages than English.

The resuting error is: 
```
[fr] REF_NOT_FOUND: Ref "/drone-plugins/drone-docker": "/Users/rkau2905/tmp/drone-plugin-index/content/banzaicloud/drone-kaniko/index.gen.fr.md:13:18"
````


Solution:
The first change is change the file generation from index.gen.<lang>.md to index_gen.<lang>.md to get a wellformed url. From this point the reference from the `rel` short code will work but link always to the default language page. To fix this, I've added a custom shortcode which uses the `relLangURL`function of hugo.
